packages/editor/src/components/collab-sidebar/comment-indicator-toolbar.js

```diff
@@ -0,0 +1,124 @@
+/**
+ * WordPress dependencies
+ */
+import { ToolbarButton } from '@wordpress/components';
+import { _x, __, sprintf } from '@wordpress/i18n';
+import { useMemo } from '@wordpress/element';
+import { privateApis as blockEditorPrivateApis } from '@wordpress/block-editor';
+
+/**
+ * External dependencies
+ */
+import clsx from 'clsx';
+
+/**
+ * Internal dependencies
+ */
+import { unlock } from '../../lock-unlock';
+
+const { CommentIconToolbarSlotFill } = unlock( blockEditorPrivateApis );
+
+const CommentAvatarIndicator = ( { onClick, thread, hasMoreComments } ) => {
+	const threadParticipants = useMemo( () => {
+		if ( ! thread ) {
+			return [];
+		}
+
+		const participantsMap = new Map();
+		const allComments = [ thread, ...thread.reply ];
+
+		// Sort by date to show participants in chronological order
```

Comment:
```suggestion
		// Sort by date to show participants in chronological order.
```
nit: I believe our style guides generally call for ending comment lines in a period.